### ISSUE TITLE
path is absolute, not relative wanted in the script lua

### ISSUE THREAD

**Issue by SbSh-DNA-ModDing** [L2KPerma/FS22_RealDashboard#2]
Error: Missing dds file 'C://gameservers//s1427066//ls2022//gameProfile/mods/_Bastel_Zeug/Dash_sdk/sdk/monitorTextures/decals.dds'
2022-10-05 23:03 Error: Can't load resource 'C://gameservers//s1427066//ls2022//gameProfile/mods/_Bastel_Zeug/Dash_sdk/sdk/monitorTextures/decals.dds'.
2022-10-05 23:03 Error: Missing dds file 'C://gameservers//s1427066//ls2022//gameProfile/mods/_Bastel_Zeug/Dash_sdk/sdk/monitorTextures/iFARM_Horizontal_screen.dds'
2022-10-05 23:03 Error: Can't load resource 'C://gameservers//s1427066//ls2022//gameProfile/mods/_Bastel_Zeug/Dash_sdk/sdk/monitorTextures/iFARM_Horizontal_screen.dds'.

**Comment by L2KPerma** [comment]
Did you copy the tablet i3d and textures to the base directory of the mod you’re adding it to or did you import the tablet i3d from the sdk folder?

**Comment by SbSh-DNA-ModDing** [comment]
yes and also edit via npp the i3d file pathes.. but it looks like an issue in the the script.
sry i've no time to fix or search it for you.  My doughter has  terminally ill with cancer.

**Comment by L2KPerma** [comment]
I along with others have installed it without issues and the script itself, shouldn’t be calling for any textures, as the textures themselves are all set up within the i3d, none are dynamically loaded via script.
